R/build-news.R

```diff
@@ -321,6 +325,16 @@ tweak_news_heading <- function(x, versions, timeline, bs_version) {
   invisible()
 }
 
+tweak_section_levels <- function(xml) {
```

Comment:
tweaking changelog section levels now to prevent having to tackle it differently when building the search index.

Reply:
Who is setting that class? pandoc?

Reply:
Yes. The changelog is written with h1 by users, but we actually want h2 now with BS4.